### PR TITLE
Add app version in settings menu on phone layout - #patch

### PR DESCRIPTION
**Pull request by pakb**
it isn't possible to read the version anywhere on phone, making it a bit hard to know if we are testing on the correct version

[Test link](https://sys-map.int.bgdi.ch/preview/fix-add-version-on-phone/index.html)

### CODE REVIEW

File: src/modules/menu/components/settings/MenuSettings.vue [hunk 1]
```diff
@@ -1,6 +1,13 @@
 <script setup>
+import { computed } from 'vue'
+import { useStore } from 'vuex'
+
 import LangSwitchToolbar from '@/modules/i18n/components/LangSwitchToolbar.vue'
+import MapFooterAppVersion from '@/modules/map/components/footer/MapFooterAppVersion.vue'
```
Comment (ltshb) [inline src/modules/menu/components/settings/MenuSettings.vue:6]: Not a good idea to do cross module import !

File: src/modules/menu/components/settings/MenuSettings.vue [hunk 2]
```diff
@@ -11,5 +18,6 @@ import LinksToolbar from '@/modules/menu/components/settings/LinksToolbar.vue'
         <div id="menu-links">
             <LinksToolbar />
         </div>
+        <MapFooterAppVersion v-if="isPhoneMode" />
```
Comment (ltshb) [inline src/modules/menu/components/settings/MenuSettings.vue:21]: I would not place the version on mobile PROD here, it is too predominant ! I know this is helpful for developers but not for the users. You can maybe try to keep it in the footer on mobile in the center very small and in white that we almost don't see it.

**Review by ltshb — CHANGES_REQUESTED** [review]

File: src/modules/menu/components/settings/MenuSettings.vue (hunk 1 above)
Comment (pakb) [reply src/modules/menu/components/settings/MenuSettings.vue]: I moved it to shared components

File: src/modules/menu/components/settings/MenuSettings.vue (hunk 2 above)
Comment (pakb) [reply src/modules/menu/components/settings/MenuSettings.vue]: I made it way smaller, and not take more space than before (position absolute)

File: src/modules/menu/components/settings/MenuSettings.vue [hunk 3]
```diff
@@ -1,15 +1,32 @@
 <script setup>
+import { computed } from 'vue'
+import { useStore } from 'vuex'
+
 import LangSwitchToolbar from '@/modules/i18n/components/LangSwitchToolbar.vue'
 import LinksToolbar from '@/modules/menu/components/settings/LinksToolbar.vue'
+import MapFooterAppVersion from '@/utils/components/AppVersion.vue'
```
Comment (ltkum) [inline src/modules/menu/components/settings/MenuSettings.vue:7]: I would import `AppVersion` here, unless there is something I'm completely missing.

File: src/views/MapView.vue [hunk 4]
```diff
@@ -17,6 +16,7 @@ import TimeSliderButton from '@/modules/map/components/toolbox/TimeSliderButton.
 import MapModule from '@/modules/map/MapModule.vue'
 import MenuModule from '@/modules/menu/MenuModule.vue'
 import { UIModes } from '@/store/modules/ui.store'
+import MapFooterAppVersion from '@/utils/components/AppVersion.vue'
```
Comment (ltkum) [inline src/views/MapView.vue:19]: same as above.

**Review by ltkum — COMMENTED** [review]
I would just change the imports, otherwise looks good to me.

**Review by ltkum — APPROVED** [review]

**Review by ltshb — COMMENTED** [review]
Perso I really don't like it, it is too predominant on the mobile 
image
We should refine this before going to PROD and see with the other of the team what they think of this. On desktop it is fine as you can easily almost hide it, but here in this menu it doesn't fit in my opinion.
Also I now the usefulness to have it to debug, we don't debug that often on mobile.
If you want to have it here some other thought on how to do it
1. Make it invisible (white color) and text selectable, so you could select it to read it
2. Create a new button/link called "About", replacing the "More informations" button. This button/link on Desktop would open a window with more information like; version, application copyright, link to github repository, link to swisstopo website, ...). On mobile this could be a section in the settings menu similar to the embed section in the share menu. This would of course need some refinement in the team.
4. keep it in the footer as on desktop make it it almost invisible next to the attribution

**Comment by pakb** [comment]
replaced by #1028

**Pull request closed without merging**